lib/record_store.rb

```diff
@@ -53,7 +53,7 @@ def secrets_path
     def zones_path
       @zones_path ||= Pathname.new(
         File.expand_path(config.fetch('zones_path'),
-        File.dirname(config_path)),
```

Comment:
Not totally sure how I feel about that indentation, since we're still in the same parenthesis... 🤷

Reply:
I see what you mean but if I change the indentation there it detects as an alignment offense so I'd need to disable the `[Layout/ArgumentAlignment](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Layout/ArgumentAlignment)` cop to fix it. I suspect it's a Rubocop bug.